core/testdata/045.golden

```diff
@@ -0,0 +1,10 @@
+[
```

Comment:
Maybe you should also add a test case checking that "if a = b: ..." remains the same